test/diff.jl

```diff
@@ -33,6 +33,36 @@ dcsch = D(csch(t))
 @test isequal(expand_derivatives(D2(t)), 0)
 @test isequal(expand_derivatives(D2(5)), 0)
 
+# Vector calculus operators
+@variables vv(t,x,y)
+∇z = Gradient(z)
+∇xy = Gradient(x,y)
+Δxy = Laplacian(x,y)
+curlx = Curl(x)
```

Comment:
Curl has to be 3 dimensional?

Reply:
What's the recommended way of putting an assert into a struct?